src/PointwiseFunctions/GeneralRelativity/Christoffel.cpp

```diff
@@ -44,33 +45,37 @@ void christoffel_second_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric,
     const tnsr::AA<DataType, SpatialDim, Frame, Index>&
         inverse_metric) noexcept {
+  if (UNLIKELY(get_size(get<0, 0, 0>(*christoffel)) !=
```

Comment:
I think `destructive_resize_components` basically does all this logic and the resize for you